### PR TITLE
Common improvements and documentation

### PR DESCRIPTION
**Pull request by AlexBrad1eyCT**
Documentation additions and corrections for common entities and Psets, I think i got most of them if i missed anything please let me know

### CODE REVIEW

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Types/IfcTransportElementNonFixedTypeEnum/Documentation.md [hunk 1]
```diff
@@ -1 +1 @@
-This enumeration is used to identify **non-fixed**or **mobile**<font color="#ff0000"><b> </b></font>transport element types.
+This enumeration is used to identify **non-fixed**or **mobile** transport element types.
```
Comment (larswik) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Types/IfcTransportElementNonFixedTypeEnum/Documentation.md:1]: Maybe add a blank before "or".

**Review by larswik — CHANGES_REQUESTED** [review]

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcImpactProtectionDeviceType/DocEntity.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcImpactProtectionDeviceType" UniqueId="2611dc1c-60e0-47b7-8506-b0d8829fa389" BaseDefinition="IfcElementComponentType" EntityFlags="32">
 	<Attributes>
 		<DocAttribute Name="PredefinedType" UniqueId="be135578-d55b-4f5f-858b-5c73d8446ff4" DefinedType="IfcImpactProtectionDeviceTypeSelect">
-			<Documentation></Documentation>
+			<Documentation>Identifies the predefined type of a impact device from which the type modelled, may be set.</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcImpactProtectionDeviceType/DocEntity.xml:5]: +1

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcPavement/DocEntity.xml [hunk 3]
```diff
@@ -2,7 +2,8 @@
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcPavement" UniqueId="806e6bf1-7065-41a4-82d3-6e15bdeba9f4" BaseDefinition="IfcBuiltElement" EntityFlags="32">
 	<Attributes>
 		<DocAttribute Name="PredefinedType" UniqueId="e9ca33dd-7f37-4b9b-9231-d75e23a6aa69" DefinedType="IfcPavementTypeEnum" AttributeFlags="1">
-			<Documentation>Boolean to identify the pavement type as a Flexible or Rigid structure.</Documentation>
+			<Documentation>Identifies the predefined type of a pavement element from which the type modelled, may be set. This type may associate additional specific property sets.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcPavement/DocEntity.xml:5]: +1

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcPavementType/DocEntity.xml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcPavementType" UniqueId="1bae4b09-c65e-42c9-8806-9ebedd5ed98d" BaseDefinition="IfcBuiltElementType" EntityFlags="32">
 	<Attributes>
 		<DocAttribute Name="PredefinedType" UniqueId="f13a48d7-7e1f-4330-a2cb-f5818f3fe02c" DefinedType="IfcPavementTypeEnum">
-			<Documentation>Boolean to identify the pavement type as a Flexible or Rigid structure.</Documentation>
+			<Documentation>Identifies the predefined type of a pavement element from which the type modelled, may be set.</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcPavementType/DocEntity.xml:5]: +1

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcSign/DocEntity.xml [hunk 5]
```diff
@@ -2,7 +2,8 @@
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcSign" UniqueId="4be0513f-edaf-4911-92c7-421ea6cd62a3" BaseDefinition="IfcElementComponent" EntityFlags="32">
 	<Attributes>
 		<DocAttribute Name="PredefinedType" UniqueId="74f12d58-b609-4a37-987a-3cae7eace471" DefinedType="IfcSignTypeEnum" AttributeFlags="1">
-			<Documentation></Documentation>
+			<Documentation>Identifies the predefined type of a signs from which the type modelled, may be set. This type may associate additional specific property sets.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcSign/DocEntity.xml:5]: +1

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcSignType/DocEntity.xml [hunk 6]
```diff
@@ -2,7 +2,7 @@
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcSignType" UniqueId="b8d00ea4-c9e5-4f74-ab2a-d8235b911718" BaseDefinition="IfcElementComponentType" EntityFlags="32">
 	<Attributes>
 		<DocAttribute Name="PredefinedType" UniqueId="c1a63671-2264-48b4-9681-6f07cadc71db" DefinedType="IfcSignTypeEnum">
-			<Documentation></Documentation>
+			<Documentation>Identifies the predefined type of a signs from which the type modelled, may be set.</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcSignType/DocEntity.xml:5]: +1

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcSignal/DocEntity.xml [hunk 7]
```diff
@@ -2,7 +2,8 @@
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcSignal" UniqueId="15911371-83a2-4660-b0a2-b479e9560615" BaseDefinition="IfcFlowTerminal" EntityFlags="32">
 	<Attributes>
 		<DocAttribute Name="PredefinedType" UniqueId="757c31d1-db40-4b9b-8140-a9f6d36a144a" DefinedType="IfcSignalTypeEnum" AttributeFlags="1">
-			<Documentation></Documentation>
+			<Documentation>Identifies the predefined type of a signal from which the type modelled, may be set. This type may associate additional specific property sets.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcSignal/DocEntity.xml:5]: +1

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcSignalType/DocEntity.xml [hunk 8]
```diff
@@ -2,7 +2,8 @@
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcSignalType" UniqueId="abb9523d-45c0-45dd-9e66-2bd5b046e9fe" BaseDefinition="IfcFlowTerminalType" EntityFlags="32">
 	<Attributes>
 		<DocAttribute Name="PredefinedType" UniqueId="8173696b-32d0-4c32-9107-403c711a0ec2" DefinedType="IfcSignalTypeEnum">
-			<Documentation></Documentation>
+			<Documentation>Identifies the predefined type of a signal from which the type modelled, may be set.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcSignalType/DocEntity.xml:5]: +1

**Review by SergejMuhic — COMMENTED** [review]
More than half of my comments refer to the PredefinedType description. No worries, just a suggestion to simplify. :)

File: IFC4x3/Constants/b/BIQUADRATICPARABOLA_14V3RUc6LFh9bZo1rEFhxL.xml [hunk 9]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BIQUADRATICPARABOLA_14V3RUc6LFh9bZo1rEFhxL" Name="BIQUADRATICPARABOLA" UniqueId="447c36de-9865-4fac-9963-c81d4e3ebed5">
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="BIQUADRATICPARABOLA" UniqueId="447c36de-9865-4fac-9963-c81d4e3ebed5">
```
[the remark this answers is not in the document]
Comment (AlexBrad1eyCT) [reply IFC4x3/Constants/b/BIQUADRATICPARABOLA_14V3RUc6LFh9bZo1rEFhxL.xml:2]: sorted

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Types/IfcTransportElementNonFixedTypeEnum/Documentation.md (hunk 1 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Types/IfcTransportElementNonFixedTypeEnum/Documentation.md:1]: nice catch fixed

File: IFC4x3/Constants/l/LONGITUDINAL_0k4SnEIKP3eQNUDPFSWApR.xml [hunk 10]
```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LONGITUDINAL_0k4SnEIKP3eQNUDPFSWApR" Name="LONGITUDINAL" UniqueId="2e11cc4e-4946-43a1-a5de-3593dc80acdb" />
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LONGITUDINAL_0k4SnEIKP3eQNUDPFSWApR" Name="LONGITUDINAL" UniqueId="2e11cc4e-4946-43a1-a5de-3593dc80acdb">
+	<Documentation>A longitidunal usage implies a subdivision along a linear facility where the part hasa defined start and end along the facility alignment(s).</Documentation>
```
[the remark this answers is not in the document]
Comment (AlexBrad1eyCT) [reply IFC4x3/Constants/l/LONGITUDINAL_0k4SnEIKP3eQNUDPFSWApR.xml:3]: sorted

File: IFC4x3/PropertyConstants/d/DEMOLISH_1ImWWTgxn0Rx9Wgwk3G9XM.xml [hunk 11]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="DEMOLISH_1ImWWTgxn0Rx9Wgwk3G9XM" Name="DEMOLISH" UniqueId="52c2081d-abbc-406f-b260-abab83409856" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="DEMOLISH" UniqueId="52c2081d-abbc-406f-b260-abab83409856" />
```
[the remark this answers is not in the document]
Comment (AlexBrad1eyCT) [reply IFC4x3/PropertyConstants/d/DEMOLISH_1ImWWTgxn0Rx9Wgwk3G9XM.xml:2]: sorted

File: IFC4x3/PropertyConstants/e/EXISTING_0zWzHkBQH9KPgJBhLqZc5Z.xml [hunk 12]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="EXISTING_0zWzHkBQH9KPgJBhLqZc5Z" Name="EXISTING" UniqueId="3d83d46e-2da4-4951-9a93-2eb5748e6163" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="EXISTING" UniqueId="3d83d46e-2da4-4951-9a93-2eb5748e6163" />
```
[the remark this answers is not in the document]
Comment (AlexBrad1eyCT) [reply IFC4x3/PropertyConstants/e/EXISTING_0zWzHkBQH9KPgJBhLqZc5Z.xml:2]: sorted

File: IFC4x3/Constants/r/REGION_1UCtBdxyD3IhR0nOLSW6Tb.xml [hunk 13]
```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="REGION_1UCtBdxyD3IhR0nOLSW6Tb" Name="REGION" UniqueId="5e3372e7-efc3-434a-b6c0-c5855c806765" />
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="REGION_1UCtBdxyD3IhR0nOLSW6Tb" Name="REGION" UniqueId="5e3372e7-efc3-434a-b6c0-c5855c806765">
+	<Documentation>A regional usage implies a bounded subdivision in the &quot;X-Y plane&quot; of a larger facility or facility part. the larger facility can be longitudinal or regional in nature.</Documentation>
```
[the remark this answers is not in the document]
Comment (AlexBrad1eyCT) [reply IFC4x3/Constants/r/REGION_1UCtBdxyD3IhR0nOLSW6Tb.xml:3]: updated to 'parent IfcSpatialStructureElement'.

File: IFC4x3/PropertyConstants/n/NEW_2Mmm0WNuPEsxi1hfv0PzYn.xml [hunk 14]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NEW_2Mmm0WNuPEsxi1hfv0PzYn" Name="NEW" UniqueId="96c30020-5f86-4edb-bb01-ae9e4067d8b1" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="NEW" UniqueId="96c30020-5f86-4edb-bb01-ae9e4067d8b1" />
```
[the remark this answers is not in the document]
Comment (AlexBrad1eyCT) [reply IFC4x3/PropertyConstants/n/NEW_2Mmm0WNuPEsxi1hfv0PzYn.xml:2]: sorted

File: IFC4x3/PropertyConstants/t/TEMPORARY_3MpsFaeAD33h8OSkOafpSq.xml [hunk 15]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TEMPORARY_3MpsFaeAD33h8OSkOafpSq" Name="TEMPORARY" UniqueId="d6cf63e4-a0a3-430e-b218-72e624a73734" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="TEMPORARY" UniqueId="d6cf63e4-a0a3-430e-b218-72e624a73734" />
```
[the remark this answers is not in the document]
Comment (AlexBrad1eyCT) [reply IFC4x3/PropertyConstants/t/TEMPORARY_3MpsFaeAD33h8OSkOafpSq.xml:2]: sorted

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcBuiltSystem/DocEntity.xml [hunk 16]
```diff
@@ -2,7 +2,7 @@
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcBuiltSystem" UniqueId="82d4f6fb-fd80-404d-96f2-af2fb5c64e6c" BaseDefinition="IfcSystem" EntityFlags="32">
 	<Attributes>
 		<DocAttribute Name="PredefinedType" UniqueId="091e3031-c454-4c42-9685-37ac7c4538ab" DefinedType="IfcBuiltSystemTypeEnum" AttributeFlags="1">
-			<Documentation></Documentation>
+			<Documentation>Identifies the predefined type of the system from which the type modelled, may be set. This type may associate additional specific property sets.</Documentation>
```
[the remark this answers is not in the document]
Comment (AlexBrad1eyCT) [reply IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcBuiltSystem/DocEntity.xml:5]: sure, this is copied from the UML and originally copied from the last IFC release. i Simplifiied see new commit

**Comment by AlexBrad1eyCT** [comment]
added requested changes

**Pull request merged**